frontend/pages/signin.tsx

```diff
@@ -11,11 +11,12 @@ export default function SignIn() {
     }[]
   >([]);
   const router = useRouter();
+  const baseUrl = process.env.NEXT_PUBLIC_BASE_URL;
```

Comment:
Maybe it would be better to use another name both here and in the env file, because it is really the base url of the API.

Reply:
Done